src/current/_includes/v24.2/backups/skip-localities-check-serverless.md

```diff
@@ -0,0 +1 @@
+Restoring a multi-region cluster into a single-region CockroachDB {{ site.data.products.serverless }} cluster is not supported. It is not possible to use the `skip-localities-check` option with `RESTORE`, because a {{ site.data.products.serverless }} cluster does not allow you to manually reconcile any restored regions that the cluster cannot meet. To restore a multi-region cluster, you must [create a new multi-region]({% link cockroachcloud/serverless-cluster-management.md %}#move-cluster-to-a-new-region) {{ site.data.products.serverless }} cluster with the desired regions that match the backup.
```

Comment:
this link would be easier to read IMO if it included all of the 'create a new MR Serverless cluster' text, but idk if that works with the Serverless include tag so ... pls ignore if so :-}